website/addons/wiki/templates/edit.mako

```diff
@@ -39,15 +51,9 @@
            </div>
         </div>
 
-        <!-- Menu without toggle in XS size only -->
-        <div class="osf-panel visible-xs">
-            <div class="osf-panel-header"> <i class="fa fa-list"> </i>  Menu </div>
-            <div class="osf-panel-body ">
-                <%include file="wiki/templates/toc.mako"/>
-            </div>
-        </div>
```

Comment:
I removed the narrow size panel because it used the ```toc.mako``` template, which I have removed. On narrow screens, the menu looks like this: 
![screen shot 2015-06-12 at 3 20 13 pm](https://cloud.githubusercontent.com/assets/7913604/8138191/96330744-1116-11e5-87aa-ec1e2fb8381f.png)